.github/workflows/ci.yml

```diff
@@ -20,187 +20,149 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  test:
+  lint:
+    name: Lint
     runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Node.js {{ matrix.node-version }}
+        uses: actions/setup-node@v3
```

Comment:
```suggestion
        uses: actions/setup-node@v4
```

Reply:
Sorry, not seeing where this was resolved. Could we link the convo here on why we are not using the latest for this?

Reply:
Oops, good catch. Updated to v4 for real for real